app/messages.py

```diff
@@ -147,7 +147,7 @@
     "message": "A user with that " "email already exists."
 }
 USER_IS_NOT_REGISTERED_IN_THE_SYSTEM = {
-    "message": "You are not registered in" "the system."
+    "message": "You are not registered in" " the system."
```

Comment:
Please change message `"You are not registered in" " the system" to "You are not registered in the system".

Reply:
ok